features/correction/service.py

```
@@ -0,0 +1,215 @@
+
+        if correction.get("status") != CorrectionStatus.NOT_STARTED.value:
+            return
+
+        await self._repository.update_status(correction_id, CorrectionStatus.DOING_RAG.value)
+        background_tasks.add_task(self._run_rag, correction_id)
+
+    async def _run_rag(self, correction_id: str) -> None:
+        """RAG 실행 후 기업 인사이트를 저장"""
+        correction = await self._repository.get_by_id(correction_id)
+        if correction is None:
+            raise ValueError(f"첨삭을 찾을 수 없습니다: {correction_id}")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)